opendevin/events/action/message.py

```diff
@@ -18,4 +19,7 @@ def message(self) -> str:
     def __str__(self) -> str:
         ret = f'**MessageAction** (source={self.source})\n'
         ret += f'CONTENT: {self.content}'
+        if self.images_base64:
+            for url in self.images_base64:
+                ret += f'\nIMAGE_URL: {url}'
```

Comment:
Can you please tell, is `images_base64` a list of URLs, e.g. a list of "https://stuff.com/img.png"? It seems so, then can we name it `images_urls`, like the list of URLs is named elsewhere?

Reply:
base64 encoded images are send from the frontend, but they can also be just urls if needed. what should i call them then.

Reply:
i will name it images_url